### PR TITLE
[WIP]Hands... The Spaghetti won't come off!

### PR DESCRIPTION
**Pull request by IK3I**
Early stages of my latest attempt to make hand code not induce vomiting. I'll throw more info up here as I start going places but you can look at initial commits to see the current progress and whatnot.

## Priorities
* Add comments to improve readability
* Remove redundant and deprecated methods
* Improve implementation where possible
* Reduce bloat and bring up to a respectable standard

#### drop_v is the first casualty of the scrubbing. 
It just does a stat check then calls drop, which is pointless because if stat was true, the items would either be dropped already or they would have nodrop...
#### put_in_any_hand_if_possible() is the next on the chopping block.
Every usage was identical to put_in_hands() so it was replaced and deleted as another deprecated method.
#### drop_items() is the first new addition to the code
```
drop_l_hand()
drop_r_hand()
```
Is a common sight in the current version, so it was condensed into a single function.
## TLDR
* drop_v() gone
* put_in_any_hand_if_possible() gone
* drop_items() added

### CODE REVIEW

**Comment by uraniummeltdown** [comment]
It would be better if you ported /tg/'s inventory code refactors

**Comment by IK3I** [comment]
you're gonna need to provide links to the refactors you're referring to. Porting isn't always the best route, and a quick search of merged tg prs didn't show anything particularly compelling.

**Comment by IK3I** [comment]
I'll start with 20283. That one seems reasonable and while a bit more grand in scope, pretty much does what I was gonna do anyway. We'll save 22918 and 31386 for later.
